pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArrayFunctions.java

```diff
@@ -151,4 +151,29 @@ public static double[] arrayConcatDouble(double[] values1, double[] values2) {
   public static String[] arrayConcatString(String[] values1, String[] values2) {
     return ArrayUtils.addAll(values1, values2);
   }
+
+  @ScalarFunction
+  public static int arrayElementAtInt(int[] arr, int idx) {
```

Comment:
We might want to support negative index (from the last element)

Reply:
SQL standard element access starts with 1, which is a bit confusing with negative indexing in most language. let's leave this out for now. so although this is a pretty common usage